TCAT/Controllers/FavoritesViewController.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  FavoritesViewController.swift
+//  TCAT
+//
+//  Created by Lucy Xu on 11/4/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import Pulley
+import UIKit
+
+class FavoritesViewController: UIViewController {
+
+    // MARK: - View vars
+    private var editButton: UIButton!
+    private let favoritesBlueColor = UIColor(hex: "08A0E0")
+    private var favoritesCollectionView: UICollectionView!
+    private var favoritesTitleLabel: UILabel!
+    private let tab = UIView()
+    private let tabSize = CGSize(width: 32, height: 4)
+
+    // MARK: - Data vars
+    private let editTitle = "Edit"
+    private let favoritesTitle = "Favorites"
+    private let favoritesReuseIdentifier = "FavoritesCollectionViewCell"
+    private var favorites: [String] = ["Collegetown Bagels", "Collegetown Bagels", "Collegetown Bagels", "Collegetown Bagels"] // Temp
+
+    override func viewDidLoad() {
+
+        super.viewDidLoad()
+        setupLabels()
+        setupFavoritesCollectionView()
+        setupTab()
+        setupConstraints()
+
+    }
+
+    private func setupLabels() {
+        favoritesTitleLabel = UILabel()
+        favoritesTitleLabel.text = favoritesTitle
+        favoritesTitleLabel.textColor = .black
+        favoritesTitleLabel.font = .getFont(.medium, size: 24)
+        view.addSubview(favoritesTitleLabel)
+
+        editButton = UIButton()
+        let attributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont.getFont(.regular, size: 14.0),
+            .foregroundColor: favoritesBlueColor,
+        ]
+        let attributedString = NSMutableAttributedString(string: editTitle, attributes: attributes)
+        editButton.setAttributedTitle(attributedString, for: .normal)
+        view.addSubview(editButton)
```

Comment:
did you mean to put this in a different function? its a button but function name is `setupLabels()`